### ISSUE TITLE
Inline Code should not be encoded

### ISSUE THREAD

**Issue by RickStrahl** [mysticmind/reversemarkdown-net#33]
If inline code blocks in single ticks contain special HTML characters the inline code is not HTML decoded properly.

```cs
[Fact]
public void When_InlineCode_Shouldnt_Contain_Encoded_Chars()
{
    var html = @"This is inline code: <code>&lt;AspNetCoreHostingModel&gt;</code>.";
    var expected = @"This is inline code: `<AspNetCoreHostingModel>`.";

    var converter = new Converter();
    var result = converter.Convert(html);
    Assert.Equal(expected, result, StringComparer.OrdinalIgnoreCase);
}
```

```
This is inline code: <code>&lt;AspNetCoreHostingModel&gt;</code>.
```

renders as:

```
This is inline code: `&lt;AspNetCoreHostingModel&gt;`.";
```

but should render as:

```
This is inline code: `<AspNetCoreHostingModel>`.";
```

**Comment by mysticmind** [comment]
Closed via #34

**Comment by mysticmind** [comment]
Available in v3.4.0
